src/amo/components/Footer/index.js

```diff
@@ -173,8 +173,6 @@ export class FooterBase extends React.Component {
                   className="Footer-lockwise-link"
                   href={`https://www.mozilla.org/firefox/lockwise/${makeQueryStringWithUTM(
                     {
-                      utm_source: 'addons.mozilla.org',
-                      utm_medium: 'referral',
```

Comment:
Are these not been added here because they're added automatically somewhere else now?

Reply:
they are default values of `makeQueryStringWithUTM`. I guess this file was updated by a contributor and we didn't notice :p